src/UBAFeeCalculator/UBAFeeSpokeCalculator.ts

```diff
@@ -167,61 +172,82 @@ export default class UBAFeeSpokeCalculator {
     return this.config.getBalanceTriggerThreshold(this.chainId, this.symbol);
   }
 
+  /**
+   * A convenience method for resolving the event fee for the spoke and the given symbol and a given flow type
+   * @param amount The amount of tokens to simulate
+   * @param flowType The flow type to simulate
+   * @param flowRange The range of the flow to simulate the event for. Defaults to undefined to simulate the event for the entire flow.
+   * @returns The event fee for the spoke and the given symbol and a given flow type
+   */
+  protected getEventFee(amount: BigNumber, flowType: "inflow" | "outflow", flowRange?: UBAFlowRange): FlowFee {
+    // The rough psuedocode for this function is as follows:
+    // We'll need two inflow/outflow curves
+    // We need to determine which flow curve to use based on the flow type
+    // Compute first balancing fee <- f(x, x+amnt)
+    // Compute second balance fee (oportunity cost) <- g(x+amnt, x)
+    // Incentive Fee (LP Fee Component): first balancing fee - second balance fee
```

Comment:
I'd understood `Incentive (fee)` to be the non-LP component - i.e. the amount the actor pays to (or receives from) the penalty pot based on the action that they're taking and the balance of the SpokePool relative to its target balance.

Reply:
This line of logic was defined during a call with @nicholaspai - I'll reach out to him to verify. Alternatively, @mrice32 or @cc7768 may be able to weigh in